src/components/TextInput/TextInput.js

```diff
@@ -140,19 +146,29 @@ function PrivateTextInput({
           capitalize={capitalize}
         />
       )}
-      <input
-        type={type}
-        className={getClasses()}
-        disabled={disabled}
-        name={name}
-        placeholder={rest.placeholder}
-        onChange={onChange}
-        onBlur={onBlur}
-        value={value}
-        data-tid={rest['data-tid']}
-        aria-label={name} // https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-label_attribute
-        autoComplete={autoComplete}
-      />
+      <div className={styles.TextInputWrapper}>
+        <input
+          type={type}
+          className={getClasses()}
+          disabled={disabled}
+          name={name}
+          placeholder={rest.placeholder}
+          onChange={onChange}
+          onBlur={onBlur}
+          value={value}
+          data-tid={rest['data-tid']}
+          aria-label={name} // https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-label_attribute
+          autoComplete={autoComplete}
+        />
+        {iconPrefix && iconName && (
```

Comment:
@MayUWish you may want to do a more specific case checking here than just confirming `iconPrefix` and `iconName` both exist. E.g. if I provide random values to them that `FontAwesomeIcon` doesn't recognize, we would run into an error. My suggestion is to validate them against specific string input values.

Reply:
If it is an invalid input, then it won't display icon, instead of an error. Please see the image below ( this example is also added to markdown file as an example) Please let me know if any other action would be needed from me for this one. Thank you very much. @ChirashiSushi 

<img width="1031" alt="image" src="https://user-images.githubusercontent.com/84494313/163068447-173e4005-6bcb-481c-9a5d-c3612defe763.png">